### PR TITLE
Profile mb

### PR DESCRIPTION
**Pull request by Maxwell-Baird**
Add details about the user to the profile page.
Add a link that allows the user to change his password.
Added a test to check if when registering a new user the inputting passwords match.
Move the gem bcrypt from group development and testing to general area

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,2 +1,18 @@
 class UsersController < ApplicationController
+  def password
+  end
+
+  def update_password
+    user = User.find(current_user.id)
+    if params[:password] == params[:confirmation]
+      user.password = params[:password]
```
Comment (colintalex) [inline app/controllers/users_controller.rb:8]: I like how this was done, clean and easy to understand.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+      if user.save &&
```
Comment (DavidTTran) [inline app/controllers/users_controller.rb]: The && was changed

File: package.json [hunk 3]
```diff
@@ -3,3 +3,6 @@
   "private": true,
   "dependencies": {}
 }
+"scripts": {
+    "start": "node server.js"
+}
```
Comment (DavidTTran) [inline package.json:8]: What's this?

File: app/views/profiles/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<section id = "profile">
```
Comment (colintalex) [inline app/views/profiles/show.html.erb:1]: Clean work!

File: spec/features/profile/show_spec.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Profile' do
+  describe 'As a User' do
+    it "I see my profile page" do
+      user = User.create(name: "David", address: "123 Test St", city: "Denver", state: "CO", zip: "80204", email: "123@example.com", password: "password", role: 1)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
```
Comment (colintalex) [inline spec/features/profile/show_spec.rb:7]: This is a helpful Snippet for users, glad we found it

**Review by colintalex — APPROVED** [review]
Looks good, all the changes we wanted were done

File: package.json (hunk 3 above)
Comment (Maxwell-Baird) [reply package.json:8]: This was put in on the advice of heroku help page

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (Maxwell-Baird) [reply app/controllers/users_controller.rb]: good catch

**Pull request merged**